sia/host/contractcreation.go

```diff
@@ -190,21 +194,21 @@ func (h *Host) NegotiateContract(conn net.Conn) (err error) {
 	defer h.unlock()
 
 	// Check that the file arrived in time.
-	if h.height >= t.FileContracts[0].Start-2 {
+	if h.state.Height() >= t.FileContracts[0].Start-2 {
 		err = errors.New("file not uploaded in time, refusing to go forward with contract")
 		return
 	}
 
-	// record filename for later retrieval
-	h.files[t.FileContracts[0].FileMerkleRoot] = filename
-
 	// Submit the transaction.
 	h.transactionChan <- t
 
 	// Put the contract in a list where the host will be performing proofs of
 	// storage.
-	firstProof := t.FileContracts[0].Start + StorageProofReorgDepth
-	h.forwardContracts[firstProof] = append(h.forwardContracts[firstProof], ContractEntry{ID: t.FileContractID(0), Contract: t.FileContracts[0]})
+	co := contractObligation{
+		inConsensus: false,
+		filename:    filename,
+	}
+	h.contracts[t.FileContractID(0)] = co
```

Comment:
you can just use a literal here. It's not always the right choice, but I feel like if your variable name is `co` it should probably just be a literal.